### ISSUE TITLE
Ignoring an entire array with `excludeKeys` and `expandArrayObjects`

### ISSUE THREAD

**Issue by bmish** [mrodrig/json-2-csv#244]
## Background Information
- Module Version: `5.0.1`

The issue I'm reporting is with:
- [x] json2csv
- [ ] csv2json

I have...
- [x] searched to see if an issue has already been reported.
- [x] verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
- [x] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).

## Expected Behavior

The array listed in `excludeKeys`, and all nested keys under it, should be excluded.

## Actual Behavior

Only keys that exactly match a key in `excludeKeys` are excluded. Listing the key for the array does not work.

## Code Example
```javascript

import { json2csv } from "json-2-csv";

json2csv(
  [
    { id: 1, arr: [{ name: "foo" }] },
    { id: 2, arr: [{ name: "bar" }] },
  ],
  { expandArrayObjects: true, excludeKeys: ['arr'] },
);
```

Resulting CSV:

```csv
id,arr.name
1,foo
2,bar
```

Whereas I would prefer that `arr.name` and any other nested keys inside this array be excluded.

In a real-world use case, I have dozens of nested keys under a particular array, and I simply want to exclude everything in this one array, without having to individually list a dozen keys in `excludeKeys`.

I see that the feature for `excludeKeys` was added here:
* https://github.com/mrodrig/json-2-csv/pull/174

And it seems like the behavior with `unwindArrays` is similar to what I want:

> Note: When used with unwindArrays, arrays present at excluded key paths will not be unwound.

**Comment by mrodrig** [comment]
Hi @bmish, thanks for reporting this. That use case makes perfect sense to me and it's a great enhancement to the `excludeKeys` option. I just published `5.1.0` which should cause the output from your example to now be just:

```csv
id
1
2
```

**Comment by bmish** [comment]
Thanks a lot!
